### DISCUSSION TITLE
Numbering of additional products

### DISCUSSION THREAD

**Discussion by kempert** [pretix/pretix#3812] [Ideas]
In the order overview, additional products are not numbered consecutively, but numbers can be found under Edit products. These numbers should also be visible in the order overview to make it easier to cancel specific items.

**Comment by raphaelm** [comment]
Are you talking about backend (organizer view) or fronted (customer view)?

**Reply by kempert** [reply]
Iam Talking about the backend view.

**Comment by kempert** [comment]
One problem is that a customer cannot explicitly rebook/cancel an item, only the total quantity.

**Reply by raphaelm** [reply]
I'm not sure I follow

**Reply by kempert** [reply]
If the customer want to rebook an additional product and the product ist personalized (on different persons) he can not choose wich additional product to wich position he is deleting. he only can change the amount of products.

**Reply by raphaelm** [reply]
Ah I see, yes
